lms/static/js/edxnotes/views/note_item.js

```diff
@@ -39,6 +42,26 @@ define([
         moreHandler: function (event) {
             event.preventDefault();
             this.toggleNote();
+        },
+
+        unitLinkHandler: function (event) {
+            event.preventDefault();
+            this.logger.emit('edx.student_notes.used_unit_link', {
+                'note_id': this.model.get('id'),
+                'component_usage_id': this.model.get('usage_id')
+            }, 2000).always(_.bind(function () {
```

Comment:
Can we move this timeout to the constant `REQUEST_TIMEOUT` and use it here instead?